draft-ietf-quic-transport.md

```diff
@@ -1697,7 +1697,8 @@ connections; validating the port is therefore unlikely to be successful.
 If the client has a token received in a NEW_TOKEN frame on a previous connection
 to what it believes to be the same server, it SHOULD include that value in the
 Token field of its Initial packet.  Including a token might allow the server to
-validate the client address without an additional round trip.
+validate the client address without an additional round trip.  A client MAY use
+a token from any previous connection to that server.
```

Comment:
Do we need any caveats about how older tokens might not be able to be processed?  Or maybe tokens should have an expiry time, which would be a design change... :(

Reply:
3 paragraphs down ("Clients might receive multiple tokens on a single connection") mentions that older tokens might become invalid and that clients should send the most recent unused token. I think that covers your question.